src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -1524,19 +1521,17 @@ private string
         bool
         IsNewLine(char ch)
         {
-            bool newLine = false;
-            if (ch == '\n')
-            {
-                newLine = true;
-            }
-            else if (ch == '\r')
+            return ch == '\r' || ch == '\n';
+        }
+
+        private
```

Comment:
We
should
fix
this.

But thanks for being consistent. :)

Reply:
Can we get a good project wide reformatting done right after release? So many times I'm just following the ugly formatting around the code and then have to point that out in reviews as to why my code looks weird or doesn't follow the conventions.

Reply:
I ran the Roslyn code format tool on the code base before we went public - it didn't fix stuff like this, but we should run it again.

Reply:
Maybe just after RTM take two _formatting_ weeks to push as many formatting PRs as we can starting with Roslyn code format tool and continue with manual formatting.